mobile layout

BuenoAmigo opened the pull request:
Swiper is not added yet

Adaptive layout for mobile
added imgs for retina

BuenoAmigo committed:
mobile layout

Swiper is not added yet

Adaptive layout for mobile
added imgs for retina

BuenoAmigo committed:
updates gitignore

KulakIlya commented on src/partials/projects.html:
```diff
@@ -0,0 +1,122 @@
+              <!-- <source srcset="" media="(min-width: 768px)" />
+              <source srcset="" media="(min-width: 1440px)" /> -->
+            </picture>
+          </li>
```
Це все має бути одним li 
```suggestion
          <li class="projects-item1">
            <ul class="item1-content-list">
              <li class="content-item">#react</li>
              <li class="content-item">#js</li>
              <li class="content-item">#node js</li>
              <li class="content-item">#git</li>
            </ul>
            <div class="item1-content-text-wrapper">
              <h4 class="item1-content-text">
                Programming Across Borders: Ideas, Technologies, Innovations
              </h4>
            </div>
            <button class="projects-btn">See project</button>
            <div>
            <picture>
              <img
                class="projects-item2-img"
                src="../images/projects_img_1_1x.jpg"
                srcset="../images/projects_img_1_1x.jpg 2x"
                alt="Project image"
              />
              <source
                srcset="
                  ../images/projects_img_1_1x.webp 1x,
                  ../images/projects_img_1_1x.webp 2x
                "
              />
              <!-- <source srcset="" media="(min-width: 768px)" />
              <source srcset="" media="(min-width: 1440px)" /> -->
            </picture>
            </div>
          </li>
          
```

KulakIlya commented on src/partials/projects.html:
```diff
@@ -0,0 +1,122 @@
+                "
+              />
+              <!-- <source srcset="" media="(min-width: 768px)" />
+              <source srcset="" media="(min-width: 1440px)" /> -->
```
У тебе при 2x завантажується картинка 1x, заміни на цей код:
```suggestion
            <picture>
                <source
                  srcset="
                    ../images/projects_img_1_1x.webp 1x,
                    ../images/projects_img_1_2x.webp 2x
                  "
                />
                <img
                  class="projects-item2-img"
                  src="../images/projects_img_1_1x.jpg"
                  srcset="../images/projects_img_1_2x.jpg 2x"
                  alt="Project image"
                />
                <!-- <source srcset="" media="(min-width: 768px)" />
                <source srcset="" media="(min-width: 1440px)" /> -->
              </picture>
```

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+    margin: 0 auto;
+  }
+  .projects-title {
+    font-family: Inter Tight;
```
font-family тут лишній

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+  }
+
+  .content-item {
+    font-family: Inter Tight;
```
Тут теж font-family, слід прибрати

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+body {
+  background-color: #242527;
+}
```
Це можна прибрати,  background-color для body заданий в base.css

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+}
+
+@media screen and (max-width: 374px) {
+  .projects .container {
```
Прибери всі стилі контейнера, вони вже оголошені в base.css

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+    background-color: #ed3b44;
+    border-color: #ed3b44;
+    border-style: none;
+    font-family: Inter Tight;
```
```suggestion


```

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+
+  .item1-content-text {
+    max-height: 100%;
+    font-family: Inter Tight;
```
```suggestion


```

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+    margin: 0 auto 84px auto;
+  }
+  .projects-title {
+    font-family: Inter Tight;
```
```suggestion


```

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+  }
+
+  .content-item {
+    font-family: Inter Tight;
```
```suggestion


```

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+
+  .item1-content-text {
+    max-height: 100%;
+    font-family: Inter Tight;
```
```suggestion


```

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+    background-color: #ed3b44;
+    border-color: #ed3b44;
+    border-style: none;
+    font-family: Inter Tight;
```
```suggestion


```

KulakIlya commented on src/css/projects.css:
```diff
@@ -0,0 +1,316 @@
+    margin-bottom: 32px;
+  }
+  .item1-content-text {
+    font-family: Inter Tight;
```
```suggestion


```

BuenoAmigo committed:
Merge branch 'main' into feature/projects

BuenoAmigo committed:
mobile layout fixed

-Прибрав базові font-styles
-Картки обгорнуті в один li для -підключення swiper
-перепідключив зобрадення, щоб видавало коректне

KulakIlya commented on src/partials/projects.html:
```diff
@@ -0,0 +1,132 @@
+    <title>Document</title>
+    <link rel="stylesheet" href="../css/projects.css" />
+  </head>
+  <body>
```
Це видали

KulakIlya commented on src/partials/projects.html:
```diff
@@ -0,0 +1,132 @@
+        </button>
+      </div> -->
+    </section>
+  </body>
+</html>
```
Це також видали

KulakIlya commented on src/partials/projects.html:
```diff
@@ -0,0 +1,132 @@
+              </picture>
+            </div>
+          </li>
+        </ul>
+        <ul class="projects-list">
+          <li class="projects-item1">
+            <ul class="item1-content-list">
+              <li class="content-item">#react</li>
+              <li class="content-item">#js</li>
+              <li class="content-item">#node js</li>
+              <li class="content-item">#git</li>
+            </ul>
+            <div class="item1-content-text-wrapper">
+              <h4 class="item1-content-text">
+                Programming Across Borders: Ideas, Technologies, Innovations
+              </h4>
+            </div>
+            <button class="projects-btn">See project</button>
+            <div class="projects-item2">
+              <picture class="projects-item2-img">
+                <source
+                  srcset="
+                    ../images/projects_img_3_1x.webp 1x,
+                    ../images/projects_img_3_2x.webp 2x
+                  "
+                />
+                <img
+                  class="projects-item2-img"
+                  width="311"
+                  height="192"
+                  src="../images/projects_img_3_1x.jpg"
+                  srcset="../images/projects_img_3_2x.jpg 2x"
+                  alt="Project image"
+                />
+                <!-- <source srcset="" media="(min-width: 768px)" />
+                <source srcset="" media="(min-width: 1440px)" /> -->
+              </picture>
+            </div>
+          </li>
+        </ul>
```
Заміни код на цей, і виправ стилі. У тебе в сеції 1 ul, у якому 3 li і в кожному з li лежить 1 ul
```suggestion
        <ul class="projects-list">
          <li class="projects-item1">
            <ul class="item1-content-list">
              <li class="content-item">#react</li>
              <li class="content-item">#js</li>
              <li class="content-item">#node js</li>
              <li class="content-item">#git</li>
            </ul>
            <div class="item1-content-text-wrapper">
              <h4 class="item1-content-text">
                Programming Across Borders: Ideas, Technologies, Innovations
              </h4>
            </div>
            <button class="projects-btn">See project</button>
            <div class="projects-item2">
              <picture class="projects-item2-img">
                <source
                  srcset="
                    ../images/projects_img_1_1x.webp 1x,
                    ../images/projects_img_1_2x.webp 2x
                  "
                />
                <img
                  class="projects-item2-img"
                  width="311"
                  height="192"
                  src="../images/projects_img_1_1x.jpg"
                  srcset="../images/projects_img_1_2x.jpg 2x"
                  alt="Project image"
                />
                <source srcset="" media="(min-width: 768px)" />
                <source srcset="" media="(min-width: 1440px)" />
              </picture>
            </div>
          </li>
        </ul>
        <ul class="projects-list">
          <li class="projects-item1">
            <ul class="item1-content-list">
              <li class="content-item">#react</li>
              <li class="content-item">#js</li>
              <li class="content-item">#node js</li>
              <li class="content-item">#git</li>
            </ul>
            <div class="item1-content-text-wrapper">
              <h4 class="item1-content-text">
                Programming Across Borders: Ideas, Technologies, Innovations
              </h4>
            </div>
            <button class="projects-btn">See project</button>
            <div class="projects-item2">
              <picture class="projects-item2-img">
                <source
                  srcset="
                    ../images/projects_img_2_1x.webp 1x,
                    ../images/projects_img_2_2x.webp 2x
                  "
                />
                <img
                  class="projects-item2-img"
                  width="311"
                  height="192"
                  src="../images/projects_img_2_1x.jpg"
                  srcset="../images/projects_img_2_2x.jpg 2x"
                  alt="Project image"
                />
                <source srcset="" media="(min-width: 768px)" />
                <source srcset="" media="(min-width: 1440px)" />
              </picture>
            </div>
          </li>
        </ul>
        <ul class="projects-list">
          <li class="projects-item1">
            <ul class="item1-content-list">
              <li class="content-item">#react</li>
              <li class="content-item">#js</li>
              <li class="content-item">#node js</li>
              <li class="content-item">#git</li>
            </ul>
            <div class="item1-content-text-wrapper">
              <h4 class="item1-content-text">
                Programming Across Borders: Ideas, Technologies, Innovations
              </h4>
            </div>
            <button class="projects-btn">See project</button>
            <div class="projects-item2">
              <picture class="projects-item2-img">
                <source
                  srcset="
                    ../images/projects_img_3_1x.webp 1x,
                    ../images/projects_img_3_2x.webp 2x
                  "
                />
                <img
                  class="projects-item2-img"
                  width="311"
                  height="192"
                  src="../images/projects_img_3_1x.jpg"
                  srcset="../images/projects_img_3_2x.jpg 2x"
                  alt="Project image"
                />
                <!-- <source srcset="" media="(min-width: 768px)" />
                <source srcset="" media="(min-width: 1440px)" /> -->
              </picture>
            </div>
          </li>
        </ul>   <ul class="projects-list">
          <li class="projects-item1">
            <ul class="item1-content-list">
              <li class="content-item">#react</li>
              <li class="content-item">#js</li>
              <li class="content-item">#node js</li>
              <li class="content-item">#git</li>
            </ul>
            <div class="item1-content-text-wrapper">
              <h4 class="item1-content-text">
                Programming Across Borders: Ideas, Technologies, Innovations
              </h4>
            </div>
            <button class="projects-btn">See project</button>
            <div class="projects-item2">
              <picture class="projects-item2-img">
                <source
                  srcset="
                    ../images/projects_img_1_1x.webp 1x,
                    ../images/projects_img_1_2x.webp 2x
                  "
                />
                <img
                  class="projects-item2-img"
                  width="311"
                  height="192"
                  src="../images/projects_img_1_1x.jpg"
                  srcset="../images/projects_img_1_2x.jpg 2x"
                  alt="Project image"
                />
                <source srcset="" media="(min-width: 768px)" />
                <source srcset="" media="(min-width: 1440px)" />
              </picture>
            </div>
          </li>
          <li class="projects-item1">
            <ul class="item1-content-list">
              <li class="content-item">#react</li>
              <li class="content-item">#js</li>
              <li class="content-item">#node js</li>
              <li class="content-item">#git</li>
            </ul>
            <div class="item1-content-text-wrapper">
              <h4 class="item1-content-text">
                Programming Across Borders: Ideas, Technologies, Innovations
              </h4>
            </div>
            <button class="projects-btn">See project</button>
            <div class="projects-item2">
              <picture class="projects-item2-img">
                <source
                  srcset="
                    ../images/projects_img_2_1x.webp 1x,
                    ../images/projects_img_2_2x.webp 2x
                  "
                />
                <img
                  class="projects-item2-img"
                  width="311"
                  height="192"
                  src="../images/projects_img_2_1x.jpg"
                  srcset="../images/projects_img_2_2x.jpg 2x"
                  alt="Project image"
                />
                <source srcset="" media="(min-width: 768px)" />
                <source srcset="" media="(min-width: 1440px)" />
              </picture>
            </div>
          </li>
          <li class="projects-item1">
            <ul class="item1-content-list">
              <li class="content-item">#react</li>
              <li class="content-item">#js</li>
              <li class="content-item">#node js</li>
              <li class="content-item">#git</li>
            </ul>
            <div class="item1-content-text-wrapper">
              <h4 class="item1-content-text">
                Programming Across Borders: Ideas, Technologies, Innovations
              </h4>
            </div>
            <button class="projects-btn">See project</button>
            <div class="projects-item2">
              <picture class="projects-item2-img">
                <source
                  srcset="
                    ../images/projects_img_3_1x.webp 1x,
                    ../images/projects_img_3_2x.webp 2x
                  "
                />
                <img
                  class="projects-item2-img"
                  width="311"
                  height="192"
                  src="../images/projects_img_3_1x.jpg"
                  srcset="../images/projects_img_3_2x.jpg 2x"
                  alt="Project image"
                />
                <!-- <source srcset="" media="(min-width: 768px)" />
                <source srcset="" media="(min-width: 1440px)" /> -->
              </picture>
            </div>
          </li>
        </ul>
```

BuenoAmigo committed:
Merge branch 'main' into feature/projects

BuenoAmigo committed:
fixed

- змінено розмітку
- додано фоторафії для таблета та десктопа
- виконано адаптивну верстку для телефону та планшета
- додано функцію переходу на Гіт нашого проєкту при натисканні кнопки "See projects"
